### PR TITLE
Region recipe

### PR DESCRIPTION
**Pull request by knaaptime**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by knaaptime** [comment]
@sjsrey once region goes up to pypi, would you mind commenting here to acknowledge you'll maintain the recipe with me

File: recipes/region/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3
+
```
Comment (marcelotrevisani) [inline recipes/region/meta.yaml]: Could you please remove this extra empty line?
```suggestion
```

File: recipes/region/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    - pip
+  run:
+    - python>=3
```
Comment (marcelotrevisani) [inline recipes/region/meta.yaml]: It is necessary an empty space after ``python``
```suggestion
    - python >=3
```

**Comment by sjsrey** [comment]
> Checklist
> 
>     * [x]  License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
> 
>     * [x]  Source is from official source
> 
>     * [x]  Package does not vend other packages
> 
>     * [x]  Build number is 0
> 
>     * [x]  A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
>       recipe (see
>       [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
>       for more details)
> 
>     * [ ]  GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

I am willing to be listed here as a maintainer.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**